app/views/assessments/credentials.html.erb

```diff
@@ -0,0 +1,43 @@
+<%# For navigation breadcrumbs %>
+<% @title = "Set credentials" %>
+
+<% content_for :stylesheets do %>
+  <%= stylesheet_link_tag "assessments/create_edit" %>
+<% end %>
+
+<% content_for :javascripts do %>
+  <%= javascript_include_tag "confirm_discard_changes" %>
+<% end %>
+
+<div class="row">
+  <div class="col-md-6">
+    <%= form_for [ @course, @assessment, @cud], url: credentials_course_assessment_path(@course, @assessment) + "/", builder: FormBuilderWithDateTimeInput do |f| %>
+      <% if @course.errors.any? %>
+          <ul>
+            <% @course.errors.full_messages.each do |msg| %>
+              <li><%= msg %></li>
+            <% end %>
+          </ul>
+      <% end %>
+
+      <% if @assessment.errors.any? %>
+          <ul>
+            <% @assessment.errors.full_messages.each do |msg| %>
+              <li><%= msg %></li>
+            <% end %>
+          </ul>
+      <% end %>
+
+      <%= f.text_field :access_key_id, :placeholder => "AKIAII7NQSHQXVDNDMQQ",
```

Comment:
@ymzong Yes. It is a fake one. Just aim to give a hint about what the format should be.